src/bindings/python/src/pyopenvino/core/common.cpp

```diff
@@ -90,7 +85,10 @@ ov::element::Type get_ov_type(py::dtype& dtype) {
     if (ctype == 'U' || ctype == 'S') {
         return ov::element::string;
     }
-    return dtype_num_to_ov_type().at(dtype.num());
+    const auto dtype_map = dtype_num_to_ov_type();
+    auto ov_type_it = dtype_map.find(dtype.num());
+    OPENVINO_ASSERT(ov_type_it != dtype_map.end(), "Unsupported data type: ", dtype);
+    return ov_type_it->second;
```

Comment:
not sure why we need this change. Logically `dtype_num_to_ov_type` should care of it.

Reply:
it produce nasty map::at exception if the value is not in map and it is very tricky to find the reason, so I just add the check if the value is in the map